packages/components/src/toolbar/toolbar/style.scss

```diff
@@ -20,6 +20,12 @@
 
 .components-accessible-toolbar,
 .components-toolbar {
+
+	&[aria-orientation="vertical"] {
```

Comment:
I moved this down so that it also targets `components-toolbar` as well as `components-accessible-toolbar`